### PR TITLE
Input double trigger fix, Vi-Icon deprecation

### PR DESCRIPTION
**Pull request by jeancarlos**
| Type  | Env Vars Change |
| :---: | :---: |
| Fix | No |

## Description

* Fix: Fix double trigger of all input components
* Fix: Correction for the css style of the file input component
* Deprecation: Vi-Icon was deprecated. The "icon" options on all componentes will be refactored in the future.

### CODE REVIEW

File: docs/pages/sections.js [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
-  {
-    name: 'Tipografia e cores',
-    content: 'pages/thypography-colors.md',
-  },
-  {
-    name: 'Documentação',
-    components: '../src/components/*.vue',
-  },
+const sections = [{
+  name: 'instalacao_e_utilizacao',
```
Comment (beeblebrox3) [inline docs/pages/sections.js]: why this change? 🤔

File: src/components/Icon.vue [hunk 2]
```diff
@@ -25,6 +25,9 @@ export default {
       return `vi-icon vi-icon-${this.name}`;
     },
   },
+  mounted() {
+    console.warn('Vi-Icon is deprecated since 0.0.22.');
```
Comment (beeblebrox3) [inline src/components/Icon.vue]: Maybe we can display this message only in development mode.

File: docs/pages/sections.js (hunk 1 above)
Comment (jeancarlos) [reply docs/pages/sections.js]: Styleguidist cant deal with spaces or special characters in the title.

File: src/components/Icon.vue (hunk 2 above)
Comment (jeancarlos) [reply src/components/Icon.vue]: The idea is to show this whenever the component is used and is only shown on mout. Putting this in dev mode defeats the purpose of the warning.

Comment (beeblebrox3) [reply src/components/Icon.vue]: Take a look: https://vuejs.org/v2/guide/deployment.html

Comment (jeancarlos) [reply src/components/Icon.vue]: image
